### PR TITLE
fix vnet peering sample

### PR DESCRIPTION
**Pull request by yungezz**
## Purpose

* ...
1. add resource group creation
2. add delete action

## Does this introduce a breaking change?

- [ ] No

## Pull Request Type
What kind of change does this Pull Request introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation content changes
- [ ] Other... Please describe:

## How to Test
*  Install ansible

```bash
$ pip install ansible[azure]
```

*  [Optional] Install ansible role

```bash
$ ansible-galaxy install Azure.azure_preview_modules
$ pip install -r ~/.ansible/roles/Azure.azure_preview_modules/files/requirements-azure.txt
```

*  Get the sample playbook

```
git clone https://github.com/Azure-Samples/ansible-playbooks.git
cd ansible-playbooks
git checkout [branch-name]
```

* Test the code

```bash
ansible-playbook <filename>.yml
```

## What to Check
Verify that the playbook is successfully run.

### CODE REVIEW

File: vnet_peering.yml [hunk 1]
```diff
@@ -15,12 +15,24 @@
...
   vars:
     resource_group: "{{ resource_group_name }}"
+    resource_group_secondary: "{{ resource_group_name }}2"
     vnet_name1: "myVnet{{ rpfx }}"
     vnet_name2: "myVnet{{ rpfx }}2"
+    peering_name: peer1
     location: eastus2
-  roles:
-    - Azure.azure_preview_modules    
+  # uncomment role if you're using ansible version < 2.8.
```
Comment (zikalino) [inline vnet_peering.yml]: just have an idea to check: could we do conditional role inclusion with ansible version? (need ):

  roles:
     - role: debian_stock_config
       when: ansible_version < "2.8"

Comment (yungezz) [reply vnet_peering.yml]: no sure. and this is just a sample.

File: vnet_peering.yml [hunk 2]
```diff
@@ -15,12 +15,24 @@
...
+  # At same install, install rule by ansible-galaxy install Azure.azure_preview_modules
```
Comment (zikalino) [inline vnet_peering.yml]: Needs rewording, and let's use "azure.azure_preview_modules" lowercase.
Let me think for a while what's the best wording.....

File: vnet_peering.yml [hunk 3]
```diff
@@ -15,12 +15,24 @@
...
+  # roles:
+  #   - Azure.azure_preview_modules
```
Comment (zikalino) [inline vnet_peering.yml]: same here, lowercase

**Pull request merged**